tpm2/tpm2.go

```diff
@@ -1187,12 +1187,18 @@ func NVUndefineSpace(rw io.ReadWriter, ownerAuth string, owner, index tpmutil.Ha
 }
 
 // NVUndefineSpaceSpecial This command allows removal of a platform-created NV Index that has TPMA_NV_POLICY_DELETE SET.
-func NVUndefineSpaceSpecial(rw io.ReadWriter, index, platformPP tpmutil.Handle) error {
-	cmd, err := tpmutil.Pack(index, platformPP)
+func NVUndefineSpaceSpecial(rw io.ReadWriter, nvIndex, sessionIndex tpmutil.Handle, pAuth []byte) error {
```

Comment:
This should probably take an AuthCommand for each of the authorizations on `nvIndex` and `platform`

Reply:
I have not figured out how to get access to platform policy, when I only have the PlatformAuth, if you can help me out here I happily implement that.